app/services/prison_visits/client.rb

```diff
@@ -43,7 +43,7 @@ def request(method, route, params)
 
       response = @connection.request(options)
 
-      JSON.parse(response.body)
+      response.body.present? ? JSON.parse(response.body) : nil
```

Comment:
How about making the feedback API return an empty object rather than nil? That way we don't need to support variable type responses.